deploy/crds/logging.openshift.io_clusterloggings_crd.yaml

```diff
@@ -9,7 +9,7 @@ spec:
     listKind: ClusterLoggingList
     plural: clusterloggings
     singular: clusterlogging
-  scope: Namespaced
+  scope: ""
```

Comment:
Why do we need this change?

Reply:
auto generated

Reply:
If this is accidentally generated by using a fresh operator-sdk, please redo with operator-sdk v0.8.2. Latter is currently the version we use for CRDs.

Reply:
We don't, I suspect a codegen accident. Need to fix that.